### PR TITLE
Week 1 Web Development

### PR DESCRIPTION
**Pull request by oulily**
In this pull request, I
- added links to my social media
- added an intro about myself
- added projects I've worked on
- added pictures of conferences I've attended
- added my project ID for App Engine config

### CODE REVIEW

File: portfolio/src/main/webapp/index.html
Comment (KevinWorkman) [inline portfolio/src/main/webapp/index.html]: One thing I have to consider with my job is how my HTML will work with screen readers (for people who have low or no vision). I believe screen readers will ready out each individual letter here. If you *want* them to read out your name instead, you could use the css `letter-spacing` rule to achieve this effect: https://www.w3schools.com/cssref/pr_text_letter-spacing.asp

(To test this out, you can turn your phone's screen reader on in the accessibility settings!)

This is totally optional and up to you. It could be that you like the effect of it spelling your name, that's totally fine!

File: portfolio/src/main/webapp/index.html
Comment (KevinWorkman) [inline portfolio/src/main/webapp/index.html]: Another optional thing: Technically the Google HTML style guide recommends indenting using two spaces, and wrapping long lines: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

I personally don't have a strong opinion though, so I'm happy to leave it up to you whether you want to follow the style guide strictly or not!

File: portfolio/src/main/webapp/style.css
Comment (KevinWorkman) [inline portfolio/src/main/webapp/style.css]: Another optional style thing: the Google CSS style guide recommends indenting things by two spaces, and it recommends alphabetizing CSS declarations: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually personally disagree with that last rule, so I never do it in my personal code. (I do have to follow it when I write code for my job though!) Again, happy to leave it up to you how strictly you want to follow the style guide.

**Review by KevinWorkman — APPROVED** [review]
Left a few optional comments, but I'm happy to leave the decision up to you, so I'm approving now. Feel free to merge this in whenever you're ready!

File: portfolio/src/main/webapp/index.html
Comment (oulily) [reply portfolio/src/main/webapp/index.html]: Wow i had no idea! Thanks for letting me know:)

File: portfolio/src/main/webapp/index.html
Comment (oulily) [reply portfolio/src/main/webapp/index.html]: I'll make those changes. I like consistent spacing and good readability.

**Pull request merged**
